vector/src/main/java/fr/gouv/tchap/core/utils/RoomUtils.kt

```diff
@@ -33,21 +28,16 @@ enum class RoomTchapType {
 }
 
 object RoomUtils {
-    /**
-     * FIXME: Change roomAccessRule with real implementation
-     */
-    fun getRoomType(roomSummary: RoomSummary): RoomTchapType {
-        val roomAccessRule = RoomAccessRules.RESTRICTED
-
+    fun getRoomType(roomSummary: RoomSummary): TchapRoomType {
```

Comment:
We have to handle here the case where the room access rule is missing (roomSummary.accessRules == null), in order to handle the old rooms created before the room access rules was defined.
I advice adding a val roomAccessRule = roomSummary.accessRules at the beginning of the method to override null value (see [v1](https://github.com/dinsic-pim/tchap-android/blob/develop/vector/src/main/java/fr/gouv/tchap/util/DinsicUtils.java#L988: DIRECT in case of a direct chat without rule, RESTRICTED by default).